CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ set(_itk_build_testing ${BUILD_TESTING})
 set(_itk_build_shared ${BUILD_SHARED_LIBS})
 set(BUILD_SHARED_LIBS OFF)
 set(elastix_GIT_REPOSITORY "https://github.com/SuperElastix/elastix.git")
-set(elastix_GIT_TAG "3816950f157753ad3763cfe3988f49f788eb38c2")
+set(elastix_GIT_TAG "ccbe39b0caee91e5a50f3cca0a4659a6eba33088")
```

Comment:
It will not be valid after a rebase. If a merge is done, then it will remain valid.

Reply:
Maybe tag some version now, before switching to ITK 5.3RC3? I prefer you to switch to ITK 5.3.x sooner rather than later. If you need an excuse to do it, then yes, I really want you to do it now 😄

Reply:
Once https://github.com/SuperElastix/elastix/pull/620 is merged, we can update the git tag here.

Reply:
@dzenanz  Thanks for your patience! Here is the new git tag: https://github.com/SuperElastix/elastix/commit/a0cf5235691ab0e7416600ed29db5abf2ff8ffe2